### ISSUE TITLE
Non-standard metadata to implement

### ISSUE THREAD

**Issue by JayPanoz** [readium/readium-2#70]
Following our previous R2 engineering call, we agreed there would be non-standard metadata to support because it solves practical use cases. 

We happen to have an EPUB compatibility doc in ReadiumCSS that can serve as a reference: 

- [XML files for EPUB2 FXL](https://github.com/readium/readium-css/blob/master/docs/CSS21-epub_compat.md#interactive-and-fixed-layout-epub-2)
- [Metadata](https://github.com/readium/readium-css/blob/master/docs/CSS21-epub_compat.md#metadata)

This doc may not be exhaustive though.

**Comment by HadrienGardeur** [comment]
There's one property missing in your doc for Calibre: `calibre:title_sort`.

**Comment by JayPanoz** [comment]
Yeah saw that in the epub revision repo. Will try to add it ASAP. 

Also, there are the Kindle ones for FXL missing too – for once, they are publicly documented.

**Comment by JayPanoz** [comment]
Sigil (authoring tool) just added the [calibre meta for sorting and series](https://github.com/readium/readium-css/blob/master/docs/CSS21-epub_compat.md#calibre) in the latest release: see [log of 0.9.10](https://sigil-ebook.com/2018/07/27/sigil-0-9-10-released/)

That’s:

- `calibre:title_sort`
- `calibre:series`
- `calibre:series_index`
